pkg/storaged/index.html

```diff
@@ -658,9 +660,9 @@ <h4 class="modal-title">{{Title}}</h4>
                 {{#ComboBox}}
                 <td class="top">{{Title}}</td>
                 <td>
-                  <div data-field={{.}} class="combobox-container">
+                  <div data-field="{{.}}" class="combobox-container">
                     <div class="input-group">
-                      <input autocomplete="false" class="combobox form-control" type="text" {{#disabled}}disabled{{/disabled}} value={{Value}} />
+                      <input autocomplete="false" class="combobox form-control" type="text" {{#disabled}}disabled{{/disabled}} value="{{Value}}" />
```

Comment:
This is a actual bug fix.  The rest of the attribute values _should_ not contain any spaces ever (but it might be possible to force some into them malicously).